tailwind/tailwind.css

```
@@ -791,3 +791,14 @@ select:disabled,
   -webkit-line-clamp: 2;
   overflow: hidden;
 }
+
+/*--- Metric cards (indigo header with stat tiles) --- */
```

Stylelint (17.4.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)